### DISCUSSION TITLE
`**args` in custom metric

### DISCUSSION THREAD

**Discussion by ohenry-proda** [microsoft/FLAML#862] [Q&A]
In 
https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#optimization-metric
the inputs of the example function are as follows:
```
def custom_metric(
    X_val, y_val, estimator, labels,
    X_train, y_train, weight_val=None, weight_train=None,
    **args,
):
```
Should this be `*args`?

**Comment by ohenry-proda** [comment]
It seems this was discussed at https://github.com/microsoft/FLAML/discussions/477 but the docs were never changed, if I'm not mistaken

**Comment by skzhang1** [comment]
Thanks for your suggestion! We have made a PR to fix this mistake in the docs. https://github.com/microsoft/FLAML/pull/883.
